src/lib/openjp2/openjpeg.h

```diff
@@ -943,6 +959,12 @@ typedef struct opj_codestream_info_v2 {
     /** information regarding tiles inside image */
     opj_tile_info_v2_t *tile_info; /* FIXME not used for the moment */
 
+  /** Number of associated data boxes*/
+  OPJ_UINT32 nbasoc;
```

Comment:
technically GMLJP2 boxes are JP2 level info, not codestream-level, so it is a bit weird to put them there. @detonin any better idea of a more appropriate structure ?